### PR TITLE
Fixed #24948 -- Fixed crash when uploading bitmap images in forms.ImageField

### PR DESCRIPTION
**Pull request by coldmind**
https://code.djangoproject.com/ticket/24948

### CODE REVIEW

File: django/forms/fields.py [hunk 1]
```diff
@@ -683,7 +683,12 @@ def to_python(self, data):
 
             # Annotating so subclasses can reuse it for their own validation
             f.image = image
-            f.content_type = Image.MIME[image.format]
+            try:
+                f.content_type = Image.MIME[image.format]
```
Comment (timgraham) [inline django/forms/fields.py]: I guess we could simply use `f.content_type = Image.MIME.get(image.format)`

**Comment by coldmind** [comment]
@timgraham updated

File: docs/releases/1.8.3.txt [hunk 2]
```diff
@@ -71,3 +71,6 @@ Bugfixes
   using UUID primary keys (:ticket:`24912`).
 
 * Fixed removing ``unique_together`` constraints on MySQL (:ticket:`24972`).
+
+* Fixed crash when registering content type for valid Windows Bitmap image
```
Comment (timgraham) [inline docs/releases/1.8.3.txt]: more understandable to users: Fixed crash when uploading bitmap images in forms.ImageField

**Comment by timgraham** [comment]
Would be helpful to note in the doc note added in the original patch that the value can be `None`.

**Comment by coldmind** [comment]
@timgraham just added a note

**Comment by timgraham** [comment]
merged in cf6ce279c7671a4c83c960c5c526f92679a4fac8, thanks!

**Pull request closed without merging**